source/index.rst

```diff
@@ -0,0 +1,114 @@
+Welcome to Darker's Documentation!
+===================================
+
+|build-badge| |license-badge| |pypi-badge| |downloads-badge| |black-badge| |changelog-badge|
+
+.. |build-badge| image:: https://github.com/akaihola/darker/actions/workflows/python-package.yml/badge.svg
+   :alt: master branch build status
+   :target: https://github.com/akaihola/darker/actions/workflows/python-package.yml?query=branch%3Amaster
+.. |license-badge| image:: https://img.shields.io/badge/License-BSD%203--Clause-blue.svg
+   :alt: BSD 3 Clause license
+   :target: https://github.com/akaihola/darker/blob/master/LICENSE.rst
+.. |pypi-badge| image:: https://img.shields.io/pypi/v/darker
+   :alt: Latest release on PyPI
+   :target: https://pypi.org/project/darker/
+.. |downloads-badge| image:: https://pepy.tech/badge/darker
+   :alt: Number of downloads
+   :target: https://pepy.tech/project/darker
+.. |black-badge| image:: https://img.shields.io/badge/code%20style-black-000000.svg
+   :alt: Source code formatted using Black
+   :target: https://github.com/psf/black
```

Comment:
Why not de-duplicate these, too, using `:start-at:` and `:end-before:`?

Reply:
![image](https://github.com/user-attachments/assets/4e98acfc-1016-4b51-94c5-520eaf24f3b4)
Getting errors doing that way